tests/Fixtures/e2e/Test_Frameworks/run_tests.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+#Run test suite with phpunit
+php ../../../../bin/infection
```

Comment:
as you are re-using this I would define a readonly constant:

```bash
readonly INFECTION=../../../../bin/infection
```

Reply:
Actually maybe in order to run fully e2e tests we should use the PHAR for running those tests, what do you think?

Reply:
>as you are re-using this I would define a readonly constant:
Sounds good to me

>Actually maybe in order to run fully e2e tests we should use the PHAR for running those tests, what do you think?

\We could, but that would require building the phar, and i'm not 100% sure on how to do that, so any help would be appreciated

Reply:
I think we should think about Phar outside of this PR

Reply:
👍 I can do that when I integrate the new version of Box